exchanges/poloniex/poloniex_futures.go

```diff
@@ -567,7 +555,7 @@ func (e *Exchange) GetV3IndexPriceComponents(ctx context.Context, symbol string)
 }
 
 // GetIndexPriceKlineData obtain the K-line data for the index price.
-func (e *Exchange) GetIndexPriceKlineData(ctx context.Context, symbol string, interval kline.Interval, startTime, endTime time.Time, limit int64) (interface{}, error) {
+func (e *Exchange) GetIndexPriceKlineData(ctx context.Context, symbol string, interval kline.Interval, startTime, endTime time.Time, limit int64) (any, error) {
```

Comment:
We shouldn't really return an any here, if you can process a type to return that would be preferable